collect/central_collector.go

```diff
@@ -738,7 +735,7 @@ func (c *CentralCollector) checkAlloc() {
 		}
 		totalDataSizeSent += trace.DataSize
 		numOfTracesSent++
-		err := c.Store.WriteSpan(ctx, &centralstore.CentralSpan{TraceID: id})
+		err := c.Store.WriteSpan(ctx, &centralstore.CentralSpan{TraceID: id, IsRoot: true})
```

Comment:
How do we know this is always a root span?

Reply:
The idea here is that we want to force refinery to eject traces. The ejection mechanism is to pretend a root span has arrived and force traces starts to go through the state machine and have decision made for them

Reply:
Please add that as a code comment. It's important.